src/en/space-station-14/departments/medical/proposals/Virology Reworked.md

```diff
@@ -0,0 +1,72 @@
+
+## Contagion Tiers
+These can all stack in one virus based on volatility and mutation, and will always start with at least one of these.
+
+Blood to Blood: Requires direct blood contact to transfer, such as repeated syringe use, or the same melee weapon used against multiple people. Drinking blood may or may not count. Very low point cost.
+Saliva: Would require someone to drink or ingest the virus from an item someone else has drank or eaten off of without being cleaned first. Low point cost.
+Airborne, Inhalation: Would require people to stand near each other in order to transfer the virus. Can be blocked with breath masks. Moderate point cost.
+Contact: Virus can be picked up through contact with objects the infected has handled with a lifespan assigned to the virus for objects, as well as hugs, cuffing actions, buckling. Moderate point cost.
+Airborne, Infectious: High virulence, able to infect through anything via proximity except for full virology suits and the CMO hardsuit and Paramedic Softsuit (gotta make them useful for something). High point cost.
+All of the above can also have a species jump modifier, which will allow the virus to transfer between species and survive successfully, at a higher mutation cost.
+
+## Virus Symptoms
+Viruses should only very rarely be beneficial. They should be something that the crew should be aware of however. Viral symptoms would be generated based on the mutation point buy system with low cost mutations being most common. Ideally these symptoms would be assigned a severity as well.
+When a virus jumps species, the severity should be bumped up by one or two equivalent levels to make the event impactful. The originator species may have a minor cough while the reciever species might be literally coughing up their lungs for example.
+
+Benign: The default virus state. Doesn't do anything.
+Runny Nose/Feels Funny: Basically doesn't do anything but let the carrier know they have something with an occasional popup. At higher levels, this can manifest into diziness or drowsiness.
+Coughing/Sneezing: Makes the carrier cough or sneeze intermittently, creates an invisible area of effect for the virus to cause infection if it has a contact or airborne infection type. Should not generate an AOE if the individual is wearing a sterile mask. Severe cough can include coughing blood, and would cause minor bloodloss.
+Dizziness: Applies the drunk effect to the carrier, more severe symptoms can apply a move debuff.
+Vomiting: Causes the carrier to vomit, will create a puddle containing any viruses the patient has, and can be spread to others via drinking (why) or through contact if they don't have shoes. Can cause blood vomiting at high severity which will remove a good chunk of the affected bloodstream.
```

Comment:
I like that idea, and I think in general this would have potential for some cool interactions. Some ideas:

A janitor cleaning the puddle without using space cleaner/bleach or other antiseptics would leave the viruses in the water puddle until it evaporates, which means if anyone slips on it and "gets some of it on their skin" would likely get infected if the virus has the contact type.

And then, the mop bucket itself would become infected unless it has antiseptics in it, which means *any* puddle the janitor cleans in the near future will be contaminated by the virus.